tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "compilerOptions": {
         "baseUrl": "src",
         "jsx": "preserve",
-        "lib": ["ES2021.String"],
+        "target": "ES6",
+        "lib": ["DOM", "DOM.Iterable", "ES6", "ES2021.String"],
```

Comment:
why do we need all of these?

Reply:
Thanks for checking, we don't need all of them, I'll remove extra options